test/unit/helpers/admin/corporate_information_pages_helper_test.rb

```diff
@@ -0,0 +1,106 @@
+require "test_helper"
+
+class Admin::CorporateInformationPagesHelperTest < ActionView::TestCase
+  test "#index_table_title_row returns the title for an edition with a primary locale of :en" do
+    edition = build(:edition, title: "title")
+    assert_equal "title", index_table_title_row(edition)
+  end
+
+  test "#index_table_title_row returns the title and locale for an edition with a non-english primary locale" do
+    edition = build(:edition, title: "title", primary_locale: :fr)
+    assert_equal "title (fr)", index_table_title_row(edition)
+  end
+
+  [
+    {
+      state: :force_published,
+      expected_tag_classes: "govuk-tag govuk-tag--s govuk-tag--yellow",
+      label_text: "Force published",
+    },
+    {
+      state: :draft,
+      expected_tag_classes: "govuk-tag govuk-tag--s govuk-tag--blue",
+      label_text: "Draft",
+    },
+    {
+      state: :submitted,
+      expected_tag_classes: "govuk-tag govuk-tag--s",
+      label_text: "Submitted",
+    },
+    {
+      state: :published,
+      expected_tag_classes: "govuk-tag govuk-tag--s govuk-tag--green",
+      label_text: "Published",
+    },
+    {
+      state: :scheduled,
+      expected_tag_classes: "govuk-tag govuk-tag--s govuk-tag--turquoise",
+      label_text: "Scheduled",
+    },
+    {
+      state: :rejected,
+      expected_tag_classes: "govuk-tag govuk-tag--s govuk-tag--red",
+      label_text: "Rejected",
+    },
+    {
+      state: :withdrawn,
+      expected_tag_classes: "govuk-tag govuk-tag--s govuk-tag--grey",
+      label_text: "Withdrawn",
+    },
+    {
+      state: :unpublished,
+      expected_tag_classes: "govuk-tag govuk-tag--s govuk-tag--grey",
+      label_text: "Unpublished",
+    },
+  ].each do |hash|
+    test "#index_table_state_row returns the correct tag for the #{hash[:state]} state" do
+      state = hash[:state]
+      edition = state == :unpublished ? create(:edition, state) : build(:edition, state)
```

Comment:
This conditional _could be_ the reason why the test hasn't picked up on the "withdrawn" vs "unpublished" issue.

Does the bug present itself if this is changed to:

```suggestion
      edition = create(:edition, state)
```

?